test/fetchzip.sh fails spuriously on the Project Tester

```
[INFO] Running /var/lib/jenkins/dlang_projects/dlang/dub/test/fetchzip.sh...
Removing gitcompatibledubpackage in /var/lib/jenkins/dlang_projects/.dub/packages/gitcompatibledubpackage-1.0.2/gitcompatibledubpackage/
Removed package: 'gitcompatibledubpackage'
Removed gitcompatibledubpackage, version 1.0.2.
Removing gitcompatibledubpackage in /var/lib/jenkins/dlang_projects/.dub/packages/gitcompatibledubpackage-1.0.4/gitcompatibledubpackage/
Removed package: 'gitcompatibledubpackage'
Removed gitcompatibledubpackage, version 1.0.4.
Performing "debug" build using dmd for x86_64.
taggedalgebraic 0.10.9: building configuration "library"...
eventcore 0.8.30: building configuration "epoll"...
stdx-allocator 2.77.0: building configuration "library"...
vibe-core 1.4.0: building configuration "epoll"...
vibe-d:utils 0.8.3: building configuration "library"...
vibe-d:data 0.8.3: building configuration "library"...
vibe-d:crypto 0.8.3: building configuration "library"...
diet-ng 1.4.5: building configuration "library"...
vibe-d:stream 0.8.3: building configuration "library"...
vibe-d:textfilter 0.8.3: building configuration "library"...
vibe-d:inet 0.8.3: building configuration "library"...
vibe-d:tls 0.8.3: building configuration "openssl"...
vibe-d:http 0.8.3: building configuration "library"...
vibe-d:mail 0.8.3: building configuration "library"...
vibe-d:mongodb 0.8.3: building configuration "library"...
vibe-d:redis 0.8.3: building configuration "library"...
vibe-d:web 0.8.3: building configuration "library"...
vibe-d 0.8.3: building configuration "vibe-core"...
test_registry ~master: building configuration "application"...
Linking...
[main(----) INF] Listening for requests on http://[::]:6721/
Failed to listen on 0.0.0.0:6721
Trying to download gitcompatibledubpackage (1.0.4)
Fetching gitcompatibledubpackage 1.0.4...
[ERROR] :17 command failed
[ERROR] Script failure.
[main(----) INF] Received signal 15. Shutting down.
```

e.g. https://ci.dlang.io/blue/organizations/jenkins/dlang-org%2Fphobos/detail/PR-6280/1/pipeline

This shouldn't happen because there's no network access involved in fetching the zip package. It's stored locally. So maybe the problem is in Vibe.d's startup time.

--- a/test/fetchzip.sh
+++ b/test/fetchzip.sh
@@ -10,7 +10,7 @@ dub remove gitcompatibledubpackage --non-interactive --version=* 2>/dev/null || 
 "$DUB" build --single "$DIR"/test_registry.d
 "$DIR"/test_registry --folder="$DIR/issue1336-registry" --port=$PORT &
 PID=$!
-sleep 0.2
+sleep 1
 trap 'kill $PID 2>/dev/null || true' exit
 
 echo "Trying to download gitcompatibledubpackage (1.0.4)"

